### PR TITLE
Set default version of TFJob operator to be v1alpha2

### PR DESCRIPTION
**Pull request by kunmingg**
Fix https://github.com/kubeflow/kubeflow/issues/977

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1012)

### CODE REVIEW

**Comment by kunmingg** [comment]
/hold

Will update bootstrapper images as well once got ```lgtm```

**Comment by jlewi** [comment]
Looks good but you need to update the TFJob image.

**Comment by kunmingg** [comment]
@jlewi Need to release TFJob image for v1alpha2 first.
On [tf-operator](https://github.com/kubeflow/tf-operator), Do we have a release version or commit for v1alpha2? (Verified works)

**Comment by kunmingg** [comment]
/hold cancel
Now everything should point to v1alpha2

File: kubeflow/examples/prototypes/tf-job-simple.jsonnet [hunk 1]
```diff
@@ -10,7 +10,7 @@ local namespace = "default";
 local image = "gcr.io/kubeflow/tf-benchmarks-cpu:v20171202-bdab599-dirty-284af3";
 
 local tfjob = {
-  apiVersion: "kubeflow.org/v1alpha1",
+  apiVersion: "kubeflow.org/v1alpha2",
```
Comment (jlewi) [inline kubeflow/examples/prototypes/tf-job-simple.jsonnet]: Why did you change this? This is not correct. The other changes needed to make the spec v1alpha2. Can we just leave tf-job-simple as v1alpha1 for now.

Comment (kunmingg) [reply kubeflow/examples/prototypes/tf-job-simple.jsonnet]: Sure, let's merge this one first.
